packages/hash/frontend/src/pages/[account-slug]/entities/[entity-uuid].page.tsx

```diff
@@ -50,6 +55,16 @@ const Page: NextPageWithLayout = () => {
             } catch {
               setEntitySubgraph(undefined);
             }
+
+            const entityTypeId = getRoots(subgraph)[0]?.metadata.entityTypeId;
+
+            if (entityTypeId) {
+              const { data: typeSubgraph } = await getEntityType({
```

Comment:
Curious why we're having to do an extra roundtrip for this – can this not be resolved in the subgraph of `getEntity`?

Reply:
Wait, you're totally right. The whole `entityTypeSubgraph` is unnecessary 🤦‍♂️ . Just got ride of all. Thanks for noticing!